develop-guide/writeblitz/writeblitz.rst

```diff
@@ -246,6 +246,60 @@ other existing apis.
         function: get_traffic_stream_objects
         ...
 
+rest
+____
+
+The ``rest`` action allows you to make rest call to any endpoint on a device. Rest uses http protocols to 
```

Comment:
Rest uses http/https

Reply:
it does but, get, post, put etc are known as http methods, and https is kind of implied when you say http methods